core/README.md

```diff
@@ -39,8 +40,9 @@ You'll need:
 # on Windows
 gradlew.bat processTestResources shadowJar
 
-java -jar build/libs/osrd-all.jar \
-    simulate \
-    --config build/resources/test/tiny_infra/config_railjson.json \
-    -o sim_changelog_output.json
+# Run as service
+java -jar build/libs/osrd-all.jar api -p 8080
+
+# Check that an infra can be loaded
+java -jar build/libs/osrd-all.jar load-infra --path RAILJSON_INFRA
```

Comment:
What about specify RAILJSON_INFRA info before ?

Reply:
`RAILJSON_INFRA` is the value of the option `--path`